pysite/views/error_handlers/http_404.py

```diff
@@ -9,4 +9,4 @@ class Error404View(ErrorView):
     error_code = 404
 
     def get(self, error: NotFound):
-        return "replace me with a template, 404 not found", 404
+        return "Unable to find page", 404
```

Comment:
This change is out of scope for this PR. Also, should be templated. :P

Reply:
Yeah, sorry. I had it changed, but then I needed to revert it back, and I forgot what it originally said.

Reply:
Won't this conflict with joph's stuff now?